nautobot_circuit_maintenance/tests/test_views.py

```diff
@@ -407,3 +436,323 @@ def test_has_advanced_tab(self):
     @skip("Not implemented yet.")
     def test_get_object_anonymous(self):
         pass
+
+
+class DashboardTest(ViewTestCases.PrimaryObjectViewTestCase):
+    """View tests for CircuitMaintenance Dashboard."""
+
+    model = CircuitMaintenance
+
+    def _get_base_url(self):
+        return f"plugins:{self.model._meta.app_label}:{self.model._meta.model_name}_{{}}"
+
+    def assertInstanceEqual(self, instance, data, api=False):  # pylint: disable=arguments-differ
+        """Used to overwrite inbuilt function. Causing type issues for datetimepicker."""
```

Comment:
@chadell thoughts here? I believe that I used this from an override that you had added.

Reply:
This was added 16months ago to some issue with the datetimepicker comparison. However, maybe since then, this issue could has been solved and no longer needed. 
does it ring some bell @glennmatthews ?